lib/common.py

```diff
@@ -510,9 +523,10 @@ def snapshot(self):
 
     def checkpoint(self):
         print("Checkpoint VM")
-        return Snapshot(self.host.xe('vm-checkpoint', {'uuid': self.uuid,
-                                                     'new-name-label': '"Checkpoint of %s"' % self.uuid}),
```

Comment:
please just add the two missing spaces in the indentation of this line so that pycodestyle tolerates it :)